src/strings/csv/es.csv

```diff
@@ -710,6 +714,7 @@ PENDING,Pendiente
 PEOPLE,Personas
 PEOPLE_CARD_DESCRIPTION,¡No podemos hacerlo solos! Añada personas a su organización y asígneles funciones.
 PEOPLE_DESCRIPTION,Gestione a las personas en Terraware.
+PERCENTAGE_OF_SITE_PLANTED,{0} % del sitio
```

Comment:
The space before the % is not necessary

Reply:
created a ticket for Alex, we can get this in for now